test/conversion.test.js

```diff
@@ -27,6 +27,21 @@ describe('Curl converter should', function() {
 		});
 	});
 
+	it('[Github: #1]: remove all the $ which are present before a string method or headers', function (done) {
```

Comment:
We should have two different tests:
1. one for the exact bug fix stating that the convert function doesn't throw error when there is a $ before method
2. As you have also fixed that the values of headers and body doesn't contain a $ after importing, separate test for that too